apps/emqx_bridge_kafka/test/emqx_bridge_kafka_tests.erl

```diff
@@ -223,144 +223,136 @@ check_atom_key(Conf) when is_map(Conf) ->
+    "\n      }"
+    "\n      partition_count_refresh_interval = \"60s\""
+    "\n      partition_strategy = \"random\""
+    "\n      required_acks = \"all_isr\""
+    "\n      topic = \"test-topic-two-partitions\""
+    "\n    }"
+    "\n    local_topic = \"mqtt/local\""
+    "\n    socket_opts {"
+    "\n      nodelay = true"
+    "\n      recbuf = \"1024KB\""
+    "\n      sndbuf = \"1024KB\""
+    "\n    }"
+    "\n    ssl {enable = false, verify = \"verify_peer\"}"
+    "\n    resource_opts {"
+    "\n      health_check_interval = 10s"
+    "\n    }"
+    "\n  }"
+    "\n}".
 
-%% erlfmt-ignore
 kafka_consumer_hocon() ->
-"""
-bridges.kafka_consumer.my_consumer {
-  enable = true
-  bootstrap_hosts = \"kafka-1.emqx.net:9292\"
-  connect_timeout = 5s
-  min_metadata_refresh_interval = 3s
-  metadata_request_timeout = 5s
-  authentication = {
-    mechanism = plain
-    username = emqxuser
-    password = password
-  }
-  kafka {
-    max_batch_bytes = 896KB
-    max_rejoin_attempts = 5
-    offset_commit_interval_seconds = 3s
-    offset_reset_policy = latest
-  }
-  topic_mapping = [
-    {
-      kafka_topic = \"kafka-topic-1\"
-      mqtt_topic = \"mqtt/topic/1\"
-      qos = 1
-      payload_template = \"${.}\"
-    },
-    {
-      kafka_topic = \"kafka-topic-2\"
-      mqtt_topic = \"mqtt/topic/2\"
-      qos = 2
-      payload_template = \"v = ${.value}\"
-    }
-  ]
-  key_encoding_mode = none
-  value_encoding_mode = none
-  ssl {
-    enable = false
-    verify = verify_none
-    server_name_indication = \"auto\"
-  }
-  resource_opts {
-    health_check_interval = 10s
-  }
-}
-""".
+    "bridges.kafka_consumer.my_consumer {"
+    "\n   enable = true"
+    "\n   bootstrap_hosts = \"kafka-1.emqx.net:9292\""
+    "\n   connect_timeout = 5s"
+    "\n   min_metadata_refresh_interval = 3s"
+    "\n   metadata_request_timeout = 5s"
+    "\n   authentication = {"
+    "\n     mechanism = plain"
+    "\n     username = emqxuser"
+    "\n     password = password"
+    "\n   }"
+    "\n   kafka {"
+    "\n     max_batch_bytes = 896KB"
+    "\n     max_rejoin_attempts = 5"
+    "\n     offset_commit_interval_seconds = 3s"
+    "\n     offset_reset_policy = latest"
+    "\n   }"
+    "\n   topic_mapping = ["
+    "\n     {"
+    "\n       kafka_topic = \"kafka-topic-1\""
+    "\n       mqtt_topic = \"mqtt/topic/1\""
+    "\n       qos = 1"
+    "\n       payload_template = \"${.}\""
+    "\n     },"
+    "\n     {"
+    "\n       kafka_topic = \"kafka-topic-2\""
+    "\n       mqtt_topic = \"mqtt/topic/2\""
+    "\n       qos = 2"
+    "\n       payload_template = \"v = ${.value}\""
+    "\n     }"
+    "\n   ]"
+    "\n   key_encoding_mode = none"
+    "\n   value_encoding_mode = none"
+    "\n   ssl {"
+    "\n     enable = false"
+    "\n     verify = verify_none"
+    "\n     server_name_indication = \"auto\""
+    "\n   }"
+    "\n   resource_opts {"
+    "\n     health_check_interval = 10s"
+    "\n   }"
+    "\n }".
```

Comment:
Nit: previous version was easier to read.  :sweat_smile:

Reply:
Mostly agree. One of the issues I had is that former formatting, in addition to requiring erlfmt pragmas, was messing my editor badly for some reason. Another one is that it was kinda misleading: it doesn't work the way people usually expect when they see `"""` (at least it won't work [until Erlang/OTP 27](https://github.com/erlang/eep/blob/master/eeps/eep-0064.md)).